### PR TITLE
doc(securing-your-webhooks): show test values

### PR DESCRIPTION
**Pull request by coolaj86**
### Why:

It's always good to have test values to know that a subtle difference in implementation isn't causing buggy behavior.

### What's being changed (if available, include any code snippets, screenshots, or gifs):

Adding test values.

### Check off the following:

- [x] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline.

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Comment by cmwilson21** [comment]
@coolaj86 Thanks so much for submitting a PR! I'll get this triaged for review :zap:

**Comment by YenyenlL** [comment]
> ### Why:
> It's always good to have test values to know that a subtle difference in implementation isn't causing buggy behavior.
> 
> ### What's being changed (if available, include any code snippets, screenshots, or gifs):
> Adding test values.
> 
> ### Check off the following:
> * [x]  I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline.
>   
>   * For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
> * [x]  For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

File: content/webhooks-and-events/webhooks/securing-your-webhooks.md [hunk 1]
```diff
@@ -64,6 +64,23 @@ Your language and server implementations may differ from the following examples.
...
+
+Algorithm:       SHA-256
+Signature:       757107ea0eb2509fc211221cce984b8a37570b6d7586c22c46f4379c8b043e17
+X-Hub-Signature: sha256=757107ea0eb2509fc211221cce984b8a37570b6d7586c22c46f4379c8b043e17
+
+Algorithm:       SHA-1
+Signature:       01dc10d0c83e72ed246219cdd91669667fe2ca59
+X-Hub-Signature: sha1=01dc10d0c83e72ed246219cdd91669667fe2ca59
+```
+
```
Comment (skedwards88) [inline content/webhooks-and-events/webhooks/securing-your-webhooks.md:83]: I'm just going to break this up to present the info piece by piece. I'm also going to switch from a code block to a list, which might be slightly more accessible for screen readers.
````suggestion
Regardless of the programming language that you use to implement HMAC verification in your code, you can use the following `secret` and `payload` values to verify that your implementation is correct.

- secret: "It's a Secret to Everybody"
- payload: "Hello, World!"

If your implementation is correct and uses the SHA-256 algorithm, the signatures that you generate should match the following signature values:

- signature: 757107ea0eb2509fc211221cce984b8a37570b6d7586c22c46f4379c8b043e17
- x-hub-signature: sha256=757107ea0eb2509fc211221cce984b8a37570b6d7586c22c46f4379c8b043e17

If your implementation is correct and uses the SHA-1 algorithm, the signatures that you generate should match the following signature values:

- signature: 01dc10d0c83e72ed246219cdd91669667fe2ca59
- x-hub-signature: sha1=01dc10d0c83e72ed246219cdd91669667fe2ca59
````

**Review by skedwards88 — COMMENTED** [review]
Thanks for adding this! I'm going to modify the format a bit, then we'll get this merged for you.

File: content/webhooks-and-events/webhooks/securing-your-webhooks.md [hunk 2]
```diff
@@ -64,6 +64,22 @@ Your language and server implementations may differ from the following examples.
...
+If your implementation is correct and uses the SHA-256 algorithm, the signatures that you generate should match the following signature values:
+
+- signature: 757107ea0eb2509fc211221cce984b8a37570b6d7586c22c46f4379c8b043e17
+- x-hub-signature: sha256=757107ea0eb2509fc211221cce984b8a37570b6d7586c22c46f4379c8b043e17
+
+If your implementation is correct and uses the SHA-1 algorithm, the signatures that you generate should match the following signature values:
+
+- signature: 01dc10d0c83e72ed246219cdd91669667fe2ca59
+- x-hub-signature: sha1=01dc10d0c83e72ed246219cdd91669667fe2ca59
```
Comment (skedwards88) [inline content/webhooks-and-events/webhooks/securing-your-webhooks.md:82]:
```suggestion
- x-hub-signature: sha1=01dc10d0c83e72ed246219cdd91669667fe2ca59

```

**Review by skedwards88 — APPROVED** [review]

**Comment by Yme4u** [comment]
Sent from Yahoo Mail on Android 
 
  On Wed, Aug 9, 2023 at 8:08 PM, Sarah ***@***.***> wrote:   


@skedwards88 commented on this pull request.

Thanks for adding this! I'm going to modify the format a bit, then we'll get this merged for you.

In content/webhooks-and-events/webhooks/securing-your-webhooks.md:
> +Regardless of programming language, these values can be used to know that the implementation is correct.
+
+```yaml
+Secret:          It's a Secret to Everybody
+Payload:         Hello, World!
+
+Algorithm:       SHA-256
+Signature:       757107ea0eb2509fc211221cce984b8a37570b6d7586c22c46f4379c8b043e17
+X-Hub-Signature: sha256=757107ea0eb2509fc211221cce984b8a37570b6d7586c22c46f4379c8b043e17
+
+Algorithm:       SHA-1
+Signature:       01dc10d0c83e72ed246219cdd91669667fe2ca59
+X-Hub-Signature: sha1=01dc10d0c83e72ed246219cdd91669667fe2ca59
+```
+

I'm just going to break this up to present the info piece by piece. I'm also going to switch from a code block to a list, which might be slightly more accessible for screen readers.
⬇️ Suggested change-Regardless of programming language, these values can be used to know that the implementation is correct.
-
-```yaml
-Secret:          It's a Secret to Everybody
-Payload:         Hello, World!
-
-Algorithm:       SHA-256
-Signature:       757107ea0eb2509fc211221cce984b8a37570b6d7586c22c46f4379c8b043e17
-X-Hub-Signature: sha256=757107ea0eb2509fc211221cce984b8a37570b6d7586c22c46f4379c8b043e17
-
-Algorithm:       SHA-1
-Signature:       01dc10d0c83e72ed246219cdd91669667fe2ca59
-X-Hub-Signature: sha1=01dc10d0c83e72ed246219cdd91669667fe2ca59
-```
-
+Regardless of the programming language that you use to implement HMAC verification in your code, you can use the following `secret` and `payload` values to verify that your implementation is correct.
+
+- secret: "It's a Secret to Everybody"
+- payload: "Hello, World!"
+
+If your implementation is correct and uses the SHA-256 algorithm, the signatures that you generate should match the following signature values:
+
+- signature: 757107ea0eb2509fc211221cce984b8a37570b6d7586c22c46f4379c8b043e17
+- x-hub-signature: sha256=757107ea0eb2509fc211221cce984b8a37570b6d7586c22c46f4379c8b043e17
+
+If your implementation is correct and uses the SHA-1 algorithm, the signatures that you generate should match the following signature values:
+
+- signature: 01dc10d0c83e72ed246219cdd91669667fe2ca59
+- x-hub-signature: sha1=01dc10d0c83e72ed246219cdd91669667fe2ca59

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you are subscribed to this thread.⬇️ Suggested change

I recall Hello World, the very first time I was invited to GitHub. It's not a sect. It represented a whole new experience for me that I never knew exist. 

 For me it was like when one goes to the Grand Canyon and hollers ( Hello ) to hear the echo. 

 My apologies for the confusion and mistakes I've made. I was excited to be a part of the other side of a Field that I've been in since I was 14. I was a hardware Technician. Licensed in Louisiana at 15. I'm learning a whole new Field of Electronics. I just want to clear up my mistakes and say Thank You for your Patience.

**Comment by coolaj86** [comment]
> I'm also going to switch from a code block to a list, which might be slightly more accessible for screen readers.

We know that:
- this documentation is for programmers
- very few programmers use screen readers
- it's a net negative to make something marginally worse for the many to make it marginally better for the few
- _might be slightly more accessible_ is equivalent to _be slightly **less** accessible_

Lists are _almost_ always the worst representation for data - right behind paragraphs.

A table might be better in both cases.

A code block is acceptable.

I don't really think it needs more words. There's already a lot of duplicate information and - much like code, more sentences means more things that have to be maintained. Less is more.

That said, SEO or whatever, tomato potato.

But converting easily digestible code blocks into difficult-to-road lists... no bueno.

**Comment by eldersz** [comment]
Ччуа

**Comment by janiceilene** [comment]
@coolaj86 You can read more about GitHub's to accessibility in https://github.blog/2022-12-02-github-accessibility-and-the-disability-divide/

**Pull request merged**